components/brave_news/api/feed.idl

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
Since this is only for the brave-combined feed, and not either the internal feed representation (as defined in mojom) or the remote direct feed items, how about we name this to match the new combined_feed_parsing.cc file, so perhaps `combined_feed`?

Reply:
Done.